main.go

```diff
@@ -69,6 +70,35 @@ func main() {
 			Inventory:   &inventory.Fake{},
 			Tracker:     &announcers.Fake{},
 		}
+	} else if cfg.MinioDev {
```

Comment:
Since this stuff is mostly duplicated from the default case it'd be cleaner to pull the Pipeline up level and fill out the defaults and then specialize the stager based upon cfg.MinioDev